### PR TITLE
Improvements to elasticsearch logging docs

### PR DESCRIPTION
**Pull request by a-robinson**
Remove a dead link to the elasticsearch logging example and simplify the elasticsearch logging getting started guide.

cross-reference #548

cc @johndmulhausen

### CODE REVIEW

File: docs/getting-started-guides/logging-elasticsearch.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
 ---
 ---
 
-On the Google Compute Engine (GCE) platform the default cluster level logging support targets
-[Google Cloud Logging](https://cloud.google.com/logging/docs/) as described at the [Logging](/docs/getting-started-guides/logging) getting
-started page. Here we describe how to set up a cluster to ingest logs into Elasticsearch and view them using Kibana as an
-alternative to Google Cloud Logging.
-
-To use Elasticsearch and Kibana for cluster logging you should set the following environment variable as shown below:
+On the Google Compute Engine (GCE) platform, the default logging support targets
```
Comment (johndmulhausen) [inline docs/getting-started-guides/logging-elasticsearch.md]: Isn't this also true on Google Container Engine? (GKE)

File: docs/getting-started-guides/logging-elasticsearch.md [hunk 2]
```diff
@@ -37,8 +42,9 @@ Created [https://www.googleapis.com/compute/v1/projects/kubernetes-satnam/region
 +++ Logging using Fluentd to elasticsearch
 ```
 
-The node level Fluentd collector pods and the Elasticsearch pods used to ingest cluster logs and the pod for the Kibana
-viewer should be running in the kube-system namespace soon after the cluster comes to life.
+The per-node Fluentd pods, the Elasticsearch pods, and the Kibana pods should
+all  be running in the kube-system namespace soon after the cluster comes to
```
Comment (johndmulhausen) [inline docs/getting-started-guides/logging-elasticsearch.md]: extra space

File: docs/getting-started-guides/logging-elasticsearch.md (hunk 1 above)
Comment (a-robinson) [inline docs/getting-started-guides/logging-elasticsearch.md]: Yes, but the instructions about how to use an environment variable to change that won't work on GKE, so I'm hesitant to bring that up here. I've added a little note indicating as such.

File: docs/getting-started-guides/logging-elasticsearch.md (hunk 2 above)
Comment (a-robinson) [inline docs/getting-started-guides/logging-elasticsearch.md]: Done.

**Pull request merged**